integration/kubernetes/helm-chart/alluxio/templates/logserver/deployment.yaml

```diff
@@ -41,6 +41,19 @@ spec:
         heritage: {{ .Release.Service }}
         role: alluxio-logserver
     spec:
+      nodeSelector:
+      {{- if .Values.logserver.nodeSelector }}
+{{ toYaml .Values.logserver.nodeSelector | trim | indent 8  }}
+      {{- else if .Values.nodeSelector }}
+{{ toYaml .Values.nodeSelector | trim | indent 8  }}
+      {{- end }}
```

Comment:
I can see you choose `if`+`else if` because the other code are doing this way. However I think `if`+`if` is a better option because it allows you to have general node selectors and more specific ones. 

I think this PR (and helm chart version) is not intended for this nodeSelector change so let's remove this few lines. In a separate PR you can address this one and if you get a chance, change all node selectors to be `if` + `if`.